conjure-java-core/src/integrationInput/java/com/palantir/product/MultipleOrderedStages.java

```diff
@@ -22,7 +22,7 @@
 import javax.annotation.Generated;
 import javax.annotation.Nonnull;
 
-@JsonDeserialize(builder = MultipleOrderedStages.Builder.class)
+@JsonDeserialize(builder = MultipleOrderedStages.DefaultBuilder.class)
```

Comment:
The implementation must be used because it has the annotations. Jackson doesn't care whether it's staged or not and isn't constrained by the nice fluent API.

Reply:
Ah that's a good find 👍